app/src/main/java/com/preonboarding/sensordashboard/data/local/SensorHistoryDatabase.kt

```diff
@@ -0,0 +1,13 @@
+package com.preonboarding.sensordashboard.data.local
+
+import androidx.room.Database
+import androidx.room.RoomDatabase
+import com.preonboarding.sensordashboard.data.local.dao.SensorHistoryDao
+import com.preonboarding.sensordashboard.domain.model.SensorHistory
+
+@Database(entities = [SensorHistory::class], version = 1, exportSchema = false)
```

Comment:
```suggestion
@Database(
   entities = [SensorHistory::class], 
   version = 1, 
   exportSchema = false
)
```
줄을 바꾸면 더 보기 편할 것 같아요!